sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -60,13 +60,13 @@ export async function executeAtomXmlOperation(
     }
   } catch (err) {
     const error = new RestError(
-      `ResponseNotInAtomXMLFormat - ${err.message}`,
+      `Response not in Atom XML format`,
       RestError.PARSE_ERROR,
       response.status,
       stripRequest(response.request),
       stripResponse(response)
     );
-
+    log.error(`Error parsing response body from Service - ${err}`);
```

Comment:
log.warning.

Reply:
Updated, Service Bus didn't have a "warning" log prefix - added and updated